templates/zerver/emails/password_reset.txt

```diff
@@ -1,27 +1,26 @@
-{% if no_account_in_realm %}
-Someone (possibly you) requested a password reset email for
-{{ email }} on {{ realm_uri }}, but
-you do not have an active account in {{ realm_uri }}.
-{% if accounts %}
-{% if multiple_accounts %}
-However, you do have active accounts in the following organizations.
-{% for account in accounts %}
-{{ account.realm.uri }}
-{% endfor %}
-You can try logging in or resetting your password in the organization
-you want.
-{% else %}
-However, you do have an active account in the {{ accounts[0].realm.uri }}
-organization; you can try logging in or resetting your password there.
-{% endif %}
-{% endif %}
-{% else %}
-Psst. Word on the street is that you need a new password, {{ email }}.
+{% if account_found_in_target_realm %}
+    Psst. Word on the street is that you need a new password, {{ email }}.
+
+    It's all good. Follow the link below and we'll take care of the rest:
 
-It's all good. Follow the link below and we'll take care of the rest:
+    {{ reset_url }}
 
-{{ reset_url }}
+{% else %}
+    Someone (possibly you) requested a password reset email for
+    {{ email }} on {{ realm_uri }}, but
+    you do not have an active account in {{ realm_uri }}.
 
+    {% if other_membership_realms_uri %}
+        However, you do have active accounts in the following organizations.
+        {% for realm_uri in other_membership_realms_uri %}
+            {{ realm_uri }}
+        {% endfor %}
+        You can try logging in or resetting your password in the organization
+        you want.
+    {% elif other_membership_realm_uri %}
+        However, you do have an active account in the {{ other_membership_realm_uri }}
+        organization; you can try logging in or resetting your password there.
+    {% endif %}
```

Comment:
I think the indentation you're adding here in the plain-text template is going to make these emails look malformed (it's sorta the unfortunate reality with plain-text email that whitespace is displayed verbatim...)